docs/libcurl/curl_easy_setopt.3

```diff
@@ -182,6 +182,9 @@ Proxy port to use. See \fICURLOPT_PROXYPORT(3)\fP
 Proxy type. See \fICURLOPT_PROXYTYPE(3)\fP
 .IP CURLOPT_NOPROXY
 Filter out hosts from proxy use. \fICURLOPT_NOPROXY(3)\fP
+.IP CURLOPT_LIBPROXY
```

Comment:
The option name should describe what it does, what it's for. Not what library it uses. It might end up not using that library at some point.

Also, since this is about setting a proxy, wouldn't it be better if this used the existing proxy options somehow to also avoid the problem with more mutexed options?